### PR TITLE
improve(dependencies): upgrade prettier version

### PR DESCRIPTION
**Pull request by mrice32**
**Motivation**

To use newer typescript notation, we need a newer version of prettier (both eslint and prettier cannot be ignored).

**Summary**

Upgraded prettier and ran the formatter. Had to make a small manual fix due to a disagreement between eslint and prettier.

Note: newer versions of prettier have some different opinionated formats (like trailing commas and parens around arrow function args). We can disable some of these if we see fit. Feel free to comment if you don't like any of the new formatting.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [x]  Untested

**Issue(s)**

N/A

### CODE REVIEW

File: packages/affiliates/scripts/example-logs.js [hunk 1]
```diff
@@ -45,6 +45,4 @@ async function runTest() {
   // console.log(balances.getTokens().snapshot())
 }
 
-runTest()
```
Comment (chrismaree) [inline packages/affiliates/scripts/example-logs.js:48]: glad it no longer wraps this kind of thing

File: package.json [hunk 2]
```diff
@@ -15,8 +15,8 @@
     "load-addresses": "yarn workspace @uma/core load-addresses",
     "create-release": "lerna version --no-git-tag-version --conventional-commits --no-push",
     "publish-release": "./scripts/run_release.sh",
-    "qbuild": "yarn workspace @uma/common build && yarn workspace @uma/core build && yarn lerna run build --stream --ignore '*/*dapp*' --ignore '@uma/common' --ignore '@uma/core'",
-    "build": "yarn workspace @uma/common build && yarn workspace @uma/core build && yarn lerna run build --stream --ignore '@uma/common' --ignore '@uma/core'",
+    "qbuild": "yarn workspace @uma/core build && yarn lerna run build --stream --ignore '*/*dapp*'",
```
Comment (chrismaree) [inline package.json:18]: ha?

**Review by chrismaree — APPROVED** [review]
I scrolled through all the changes and they pretty much all seem reasonable. They are all pretty much adding an extra `,` after the last element within an object or adding `(x)` brackets around vars in functions. Few interesting changes but overall they all look good to me!

**Comment by pemulis** [comment]
Looks like this will just need to resolve some merge conflicts.

**Comment by mrice32** [comment]
Thanks for pointing this out @pemulis. Need to loop back on this!

File: .eslintrc.js [hunk 3]
```diff
@@ -2,19 +2,19 @@ module.exports = {
...
     mocha: true,
-    es2020: true
+    es2020: true,
   },
   extends: ["plugin:prettier/recommended", "eslint:recommended"],
   plugins: ["prettier", "mocha"],
   rules: {
     "prettier/prettier": ["error"],
-    indent: ["error", 2, { SwitchCase: 1 }],
+    indent: 0, // avoid conflict with prettier's indent system
```
Comment (mrice32) [inline .eslintrc.js:11]: Prettier and eslint have different implementations of indent, so even when they are configured similarly, they sometimes make conflicting decisions (encountered one in affiliates), so I turned off the eslint checker to allow the prettier one to take over.

**Pull request merged**